### ISSUE TITLE
Move index "fragmentGeneSymbol" creation to  covidgraph/graph-processing_fragmentize_text 

### ISSUE THREAD

**Issue by motey** [covidgraph/graph-processing_text_gene_match#1]
As the fulltext index for :Fragment nodes is not only relevant for gene matching it would make sense to move the creation of the index into the fragger repo at covidgraph/graph-processing_fragmentize_text and rename it from `fragmentGeneSymbol` to only `fragment`.
Tasks:

[ ] Remove index creation for fragmentGeneSymbol from covidgraph/graph-processing_text_gene_match 
[ ] Reimplement index creation in graph-processing_fragmentize_text with new name `fragment`
[ ] Check and fix any broken dependencies to `fragmentGeneSymbol`in other repos

**Comment by Jiros** [comment]
@motey - does this relate to any specific data sources? I have created an epic for each data source and we can link any related issues so that you can filter the Zenhub board by an epic and see only the relevant issues. It's simply a case of selecting the epics on the right ->

**Comment by motey** [comment]
No, its more of a processing thingy. It does not relate to any specific datasource, rather to every datasource that has large text properties.
